app/src/main/res/values-pl/strings.xml

```diff
@@ -1,203 +1,268 @@
+    <string name="NOTIFICATION_POPUPS">Wyskakujace okno powiadomień</string>
+    <string name="NOTIFICATION_POPUPS_PASSENGER">Pokaż powiadomienia z siedzącym pasażerem</string>
+    <string name="NOTIFICATION_SOUND">Odtwarzaj dźwięk powiadomienia</string>
+    <string name="NOTIFICATION_READOUT">Odczytaj powiadomienie podczas podglądu (TTS)</string>
+    <string name="NOTIFICATION_READOUT_POPUP">Odczytaj nowe powiadomienia (TTS)</string>
+    <string name="NOTIFICATION_READOUT_POPUP_PASSENGER">… z obecnym pasażerem</string>
+    <string name="READOUT_DESCRIPTION">Ta aplikacja jest używana do odczytu powiadomień (TTS)</string>
+    <string name="MAP_ACTION_VIEWMAP">Zobacz całą mapę</string>
+    <string name="MAP_ACTION_SEARCH">Szukaj miejsca</string>
+    <string name="MAP_ACTION_CLEARNAV">Wyczyść nawigacja</string>
+    <string name="MUSIC_APPLIST_TITLE">Aplikacje</string>
+    <string name="MUSIC_APPLIST_EMPTY">&lt;Brak aplikacji&gt;</string>
+    <string name="MUSIC_CUSTOMACTIONS_TITLE">Akcje</string>
+    <string name="MUSIC_DISCONNECTED">&lt;Nie połączony&gt;</string>
+    <string name="MUSIC_BROWSE_TITLE">Przeglądaj</string>
+    <string name="MUSIC_BROWSE_EMPTY">&lt;Pusto&gt;</string>
+    <string name="MUSIC_BROWSE_LOADING">&lt;Ładowanie&gt;</string>
+    <string name="MUSIC_BROWSE_SEARCHING">&lt;Wyszukiwanie&gt;</string>
+    <string name="MUSIC_BROWSE_ACTION_JUMPBACK">Powrót</string>
+    <string name="MUSIC_BROWSE_ACTION_FILTER">Filtruj</string>
+    <string name="MUSIC_BROWSE_ACTION_SEARCH">Szukaj</string>
+    <string name="MUSIC_BROWSE_PLAY_FROM_SEARCH">Odtwórz z wyszukiwania</string>
+    <string name="MUSIC_QUEUE_TITLE">Aktualnie odtwarzane</string>
+    <string name="MUSIC_QUEUE_EMPTY">&lt;Pusta kolejka&gt;</string>
+    <string name="MUSIC_SKIP_PREVIOUS">Poprzednia</string>
+    <string name="MUSIC_SKIP_NEXT">Aktualnie odtwarzane</string>
+    <string name="MUSIC_TURN_SHUFFLE_UNAVAILABLE">Odtwarzanie losowe niedostępne</string>
+    <string name="MUSIC_TURN_SHUFFLE_ON">Odtwarzanie losowe włączone</string>
+    <string name="MUSIC_TURN_SHUFFLE_OFF">Odtwarzanie losowe wyłączone</string>
+    <string name="MUSIC_TURN_REPEAT_UNAVAILABLE">Powtarzanie niedostepne</string>
+    <string name="MUSIC_TURN_REPEAT_ALL_ON">Powtarzaj wszystkie</string>
+    <string name="MUSIC_TURN_REPEAT_ONE_ON">Powtarzaj pojedynczo </string>
+    <string name="MUSIC_TURN_REPEAT_OFF">Wyłącz powtarzanie</string>
+    <string name="MUSIC_SPOTIFY_REMOVE_FROM_COLLECTION">Usuń z kolekcji</string>
+    <string name="MUSIC_SPOTIFY_START_RADIO">Radio</string>
+    <string name="MUSIC_SPOTIFY_ADD_TO_COLLECTION">Polub</string>
+    <string name="MUSIC_SPOTIFY_THUMB_UP">Kciuk w górę</string>
+    <string name="MUSIC_SPOTIFY_THUMBS_UP_SELECTED">Kciuk w górę</string>
+    <string name="MUSIC_SPOTIFY_THUMB_DOWN">Kciuk w dół</string>
+    <string name="MUSIC_SPOTIFY_THUMBS_DOWN_SELECTED">Kciuk w dół</string>
+    <string name="lbl_cds_data" />

```

Comment:
This will hide this heading, can you add a translation instead?

Reply:
lbl_cds_data - this is the experimental tab when I was playing with extra tab to present cds data. I removed it through translation manager, but somehow I was left empty in strings.xml file. You can ignore it.

Reply:
It doesn't interfere with the main app.